File: DemoApp/src/main/java/com/adform/trackingsdk/demoapp/DemoApp.java
package com.adform.trackingsdk.demoapp;

import android.app.Application;

import com.adform.adformtrackingsdk.AdformTrackingSdk;

public class DemoApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        AdformTrackingSdk.startTracking(this, Constants.TRACK_POINT_ID);
        AdformTrackingSdk.setDebugModeEnabled(true);
    }
}

File: DemoApp/src/main/kotlin/com/adform/trackingsdk/demoapp/DemoApp.kt
package com.adform.trackingsdk.demoapp

import android.app.Application
import com.adform.adformtrackingsdk.AdformTrackingSdk

class DemoApp : Application() {
    override fun onCreate() {
        super.onCreate()
        AdformTrackingSdk.startTracking(this, Constants.TRACK_POINT_ID)
        AdformTrackingSdk.setDebugModeEnabled(true)
    }
}